File: src/main/java/ru/arc/network/RedisSerializer.java
package ru.arc.network;

import ru.arc.util.Common;

public class RedisSerializer {


    public static String toJson(Object serializable) {
        return Common.gson.toJson(serializable);
    }

    public static <T> T fromJson(String json, Class<T> clazz) {
        return Common.gson.fromJson(json, clazz);
    }

}

File: src/main/kotlin/ru/arc/network/RedisSerializer.kt
package ru.arc.network

import ru.arc.util.Common

object RedisSerializer {
    @JvmStatic fun toJson(serializable: Any): String = Common.gson.toJson(serializable)
    @JvmStatic fun <T> fromJson(json: String, clazz: Class<T>): T = Common.gson.fromJson(json, clazz)
}
